tests/examples/shims/_common_template.lua

```diff
@@ -1,3 +1,4 @@
+-- luacheck: globals string
```

Comment:
yes. GitHub doesn't check if the commit changed (as it would create too many problems). This was done not long after it was reported. This line is now part of `shims: Add the string.wlen function`